packages/driver/src/cross-origin/cypress.ts

```diff
@@ -102,6 +103,15 @@ const setup = (cypressConfig: Cypress.Config, env: Cypress.ObjectLike) => {
   // @ts-ignore
   Cypress.isCy = cy.isCy
 
+  // this is valid inside the cy.origin() callback, but it should be replaced
+  // by the webpack preprocessor with an actual require() before the spec code
+  // is run in the browser. if it's not, it means the user isn't using the
+  // webpack preprocessor or is using an older version of it. this error guides
```

Comment:
can you add the version of webpack preprocessor which introduces support for this?

Reply:
I can, but it will need to wait until closer to release time in order to determine what version that will be, since there could be versions of the preprocessor released between now and when this is released.